.github/workflows/release.yml

```diff
@@ -105,20 +121,20 @@ jobs:
         with:
           script: |
             const {RELEASE_ID} = process.env
+            const {TAG_NAME} = process.env
             github.rest.repos.updateRelease({
               owner: context.repo.owner,
               repo: context.repo.repo,
               release_id: `${RELEASE_ID}`,
               draft: false,
-              tag_name: `${{ github.event.workflow_run.head_branch }}`,
-              name: `${{ github.event.workflow_run.head_branch }}`,
-              prerelease: `${{ contains(github.event.workflow_run.head_branch, '-alpha') || contains(github.event.workflow_run.head_branch, '-beta') || contains(github.event.workflow_run.head_branch, '-rc') }}`,
+              tag_name: `${TAG_NAME}`,
+              name: `${TAG_NAME}`
             });
 
       - name: Trigger chart update
-        uses: peter-evans/repository-dispatch@26b39ed245ab8f31526069329e112ab2fb224588 # v2.1.1
+        uses: peter-evans/repository-dispatch@4c03f6fcc3d786d9feef89ed5544ccf37d52e232
         with:
           token: ${{ secrets.WORKFLOW_PAT }}
           repository: "${{github.repository_owner}}/helm-charts"
           event-type: update-chart
-          client-payload: '{"version": "${{ github.event.workflow_run.head_branch }}", "oldVersion": "${{ steps.get_old_release_tag.outputs.release_tag }}", "repository": "${{ github.repository }}"}'
+          client-payload: '{"version": "${{ github.ref_name }}", "oldVersion": "${{ steps.get_last_release_tag.outputs.old_release_tag }}", "repository": "${{ github.repository }}", "crds_asset_id": "${{steps.upload_release_assets.outputs.crds_asset_id}}"}'
```

Comment:
it seems to me that this is a left-over from the controller one; we don't have `crds_asset_id` here for example. I'm not sure about the `github.ref_name` change, though.